README.md

```diff
@@ -1,7 +1,7 @@
 # [![Bevy](assets/branding/bevy_logo_light_dark_and_dimmed.svg)](https://bevyengine.org)
 
 [![Crates.io](https://img.shields.io/crates/v/bevy.svg)](https://crates.io/crates/bevy)
-[![license](https://img.shields.io/badge/license-MIT-blue.svg)](./LICENSE)
+[![license](https://img.shields.io/badge/license-MIT%2FApache-blue.svg)](./LICENSE)
```

Comment:
[![license](https://img.shields.io/badge/license-MIT%2FApache-blue.svg)](./LICENSE)

(Inline rendered for future reviewers)

For accessibility, it might be worth changing to something like:
```suggestion
[![MIT/Apache 2.0](https://img.shields.io/badge/license-MIT%2FApache-blue.svg)](./LICENSE)
```

[![MIT/Apache 2.0](https://img.shields.io/badge/license-MIT%2FApache-blue.svg)](./LICENSE)

This changes the alt text of the image - we can open an issues for the other ones if you want, or you can just fix them up in this PR.

Reply:
i dig this! what other ones?

Reply:
The full set is:
```md
[![Crates.io](https://img.shields.io/crates/v/bevy.svg)](https://crates.io/crates/bevy)
# Fine-ish, could be more descriptive
[![license](https://img.shields.io/badge/license-MIT-blue.svg)](./LICENSE)
# Updated
[![Crates.io](https://img.shields.io/crates/d/bevy.svg)](https://crates.io/crates/bevy)
# Oh no, same as above. Maybe "download count" would be better
[![Rust](https://github.com/bevyengine/bevy/workflows/CI/badge.svg)](https://github.com/bevyengine/bevy/actions)
# Hmm, not especially clear that this is a CI run
![iOS cron CI](https://github.com/bevyengine/bevy/workflows/iOS%20cron%20CI/badge.svg)
# Whilst useful to have visible, it might be worth setting up alternative reporting method and removing this. 
[![Discord](https://img.shields.io/discord/691052431525675048.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/bevy)
# Good
```
However, I have also checked, and to my layman's eye the shields.io badges are already as accessible as seems reasonable, namely having the full text as a title text. It might make more sense to not have the alt text at all. We'd need to ask someone with accessibility tools experience for their thoughts I suspect.

It's important to think about it, even if the consequences for this case are minor, but this absolutely should be spun out.